shared/src/commonTest/kotlin/com/example/tasky/auth/domain/manager/SessionManagerMock.kt

```diff
@@ -0,0 +1,105 @@
+package com.example.tasky.auth.domain.manager
+
+import com.example.tasky.auth.data.model.AccessTokenBody
+import com.example.tasky.auth.data.model.LoginResponse
+import io.ktor.client.plugins.auth.providers.BearerTokens
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import kotlin.coroutines.cancellation.CancellationException
```

Comment:
You're in your domain layer here, but make use of implementation details/external libraries (Kotlinx Serialization). In tests, I would ideally replicate the same structure as from your production code. If you introduce a class that only exists for test cases, you don't need to create any abstractions for this just to adhere to the domain isolation, but then I'd put it in a package that has nothing to do with domain, like `auth.mocks`.